include/circt/Dialect/SV/SVStatements.td

```diff
@@ -463,17 +463,20 @@ def VerbatimOp : SVOp<"verbatim"> {
 
     sv.verbatim allows operand substitutions with {{0}} syntax.
   }];
-  let arguments = (ins StrAttr:$string, Variadic<AnyType>:$operands);
```

Comment:
Plz add this to sv.verbatim.expr(.se) so they aren't artificially more limited.

Reply:
Done.

Reply:
Thanks!